app/main/views/evidence.py

```diff
@@ -60,6 +83,29 @@ def get_evidence(evidence_id):
     return jsonify(evidence=evidence_data), 200
 
 
+@main.route('/evidence/<int:evidence_id>', methods=['DELETE'])
+def delete_evidence_draft(evidence_id):
+    evidence = evidence_service.get_evidence_by_id(evidence_id)
+    if not evidence or not evidence.status == 'draft':
+        abort(400, 'Evidence is not valid or not in a draft state')
+    json_payload = get_json_from_request()
+    evidence_service.delete_evidence(evidence)
```

Comment:
Do we need the `delete_evidence` function?  Instead of calling this, can we just do `evidence_service.delete(evidence)`?